k8s/namespaces/et/kustomization.yaml

```diff
@@ -6,5 +6,3 @@ bases:
 # Warning : Adding a file here, adds to all environments to which you add your kustomization.
 - et-cos/et-cos.yaml
 - et-msg-handler/et-msg-handler.yaml
-- et-sya-api/et-sya-api.yaml
```

Comment:
Removing from here will remove et-sya app generally, I think the other two changes are enough

Reply:
Re-added here.

Reply:
Could you also re-ad the et-sya app too please, I meant to highlight both sorry